src/Editor.h

```diff
@@ -2,10 +2,14 @@
 
 #include "Scene.h"
 #include "Transform.h"
-
 namespace Editor
 {
-
+    enum GUIZMO_OPERATION_TYPE {
+        TRANSLATE,
+        ROTATE,
+        SCALE,
+        NONE
+    };
```

Comment:
Wrong indentation (if there is a namespace, we don't indent by 1 level, visual studio might be angry I know)

Let's use `enum class` instead of `enum`, it has some nicer properies https://stackoverflow.com/questions/18335861/why-is-enum-class-considered-safer-to-use-than-plain-enum

Let's use the same name convention as the rest of the code for enums and enum variants, which is ex.
```cpp
enum class DrawFunctionType
{
    Indexed,
    NotIndexed,
};
```